setup.sh

```diff
@@ -25,7 +25,7 @@ function check_installed_python() {
         exit 2
     fi
 
-    for v in 9 10 8
+    for v in 10 9 8
```

Comment:
Python 3.10 had a memory problem in january - which is why the docker-image has been downgraded to 3.9.
It's also happening on 3.10 (at least on my machine).

There's since been a ton of dependency updates, as well as new 3.10 versions - but i never had the time to verify if the problem is now gone or not.

Therefore, i will only be able to accept this if you are certain this problem is gone (and actually did some tests for it).
simply leave the bot running overnight and observe memory usage compared to a 3.9 installation with the same setup - this can be performed either via a 3.10 docker image (we don't build these at the moment) - or via a native installation.

Reply:
Seems it may be fixed. https://bugs.python.org/issue46347